python/chronos/test/bigdl/chronos/forecaster/test_pytorch.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.chronos.forecaster.pytorch import TCNForecaster, LSTMForecaster, NBeatsForecaster, Seq2SeqForecaster, AutoformerForecaster
```

Comment:
> I think it might be better to write a test class and import these forecasters in side a test_xxx method

I wonder whether all functions of those forecasters need to be tested in the test class

Reply:
> > I think it might be better to write a test class and import these forecasters in side a test_xxx method
> 
> I wonder whether all functions of those forecasters need to be tested in the test class

no, just import test should be enough.